x/ccv/provider/types/keys.go

```diff
@@ -281,6 +296,27 @@ func getKeyPrefixes() map[string]byte {
 		// minimum power required to be in the top N per consumer chain.
 		MinimumPowerInTopNKeyName: 40,
 
+		// ConsumerIdKeyName is the key for storing the consumer id for the next registered consumer chain
+		ConsumerIdKeyName: 41,
+
+		// ConsumerIdToChainIdKeyName is the key for storing the chain id for the given consumer id
+		ConsumerIdToChainIdKeyName: 42,
```

Comment:
Why do we need this. Isn't the RegistrationRecord containing the chainID?

Reply:
Good catch! Removed.